doc/Websites.md

```diff
@@ -0,0 +1,14 @@
+## Recursive links ##
```

Comment:
readme, distributed with the sources or a build could be one place

user manual would ok too, I think